config/pipeline/vars/vh/common.yaml

```diff
@@ -0,0 +1,2 @@
+variables:
+  space: holder
```

Comment:
Can we not add all these same vars to the common var file?

Reply:
We need something in the files or they report miss formatted, so I put these in just to have something in.